app/src/main/res/layout/activity_voice_over_ipactivity.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think the name of this file should conform to what we have done so far, ie., be named activity_voic_over_ip.

Reply:
Nice catch, don't know how that go there since these files are generated automatically when creating a new Activity!